src/commands/sfpowerkit/org/sandbox/refresh.ts

```diff
@@ -54,47 +62,81 @@ export default class Refresh extends SFPowerkitCommand {
       this.flags.apiversion || (await conn.retrieveMaxApiVersion());
 
     let result;
-
-    const sandboxId = await this.getSandboxId(conn, this.flags.name);
+    let [sandboxId, sandboxDescription] = await this.getSandboxDetails(conn, this.flags.name);
     const uri = `${conn.instanceUrl}/services/data/v${this.flags.apiversion}/tooling/sobjects/SandboxInfo/${sandboxId}/`;
 
     if (this.flags.clonefrom) {
-      const sourceSandboxId = await this.getSandboxId(
+      const sourceSandboxId = await this.getSandboxDetails(
         conn,
         this.flags.clonefrom
       );
-
-      result = await request({
-        method: "patch",
-        url: uri,
-        headers: {
-          Authorization: `Bearer ${conn.accessToken}`
-        },
-        body: {
-          AutoActivate: "true",
-          SourceId: `${sourceSandboxId}`
-        },
-        json: true
-      });
+    
+      if (this.flags.description) {
+        result = await request({
+          method: "patch",
+          url: uri,
+          headers: {
+            Authorization: `Bearer ${conn.accessToken}`
+          },
+          body: {
+            AutoActivate: "true",
+            SourceId: `${sourceSandboxId}`,
+            Description: `${this.flags.description}`
+          },
+          json: true
+        });
+      }else{
+        result = await request({
+          method: "patch",
+          url: uri,
+          headers: {
+            Authorization: `Bearer ${conn.accessToken}`
+          },
+          body: {
+            AutoActivate: "true",
+            SourceId: `${sourceSandboxId}`,
+            Description: `${sandboxDescription}`
```

Comment:
Can you reduce the duplication of the request by using conditional directly in the body? For eg:
```suggestion
            Description:this.flags.description?`${this.flags.description}`: `${sandboxDescription}`
```